src/store/index.js

```diff
@@ -546,6 +556,58 @@ export function sendCssToFrame(value, currentField) {
   })
 }
 
+/**
+* Function to handle migration of old variables to new variables
+* @param {String} The type of migration needed
+* @param {Object} Saved theme values
+* @return {Object} Final theme values
+*/
+export function migrateOldVariables(data) {
+  let migrated = false
+
+  for (var key in data) {
+    switch(key) {
+      case 'headingOneFont':
+        data['headingOneFontFamily'] = data[key]
+        delete data[key]
```

Comment:
`delete data[key]` and `migrated = true` are repeated over and over.

1. set `migrated=true` initially
2. on the `default` switch case block, set `migrated=false`
3. after the switch case, `if migrated` then `delete data[key]`

Reply:
@squallstar If I set the `migrated=true` initially and then the `default` switch block set to `migrated=false` means that if the loop ends in a key that doesn't match any of the cases it will turn the `migrated=false` and then the return that the data wasn't migrated.

I need to know when changes happened to the object.